core-kotlin/build.gradle.kts

```diff
@@ -26,6 +26,7 @@ dependencies {
     implementation(dependency.kotlin.coroutines)
     implementation(dependency.androidx.core.ktx)
     implementation(project(":core"))
+    implementation(project(":common-core"))
```

Comment:
Is this still needed since core pulls in :common-core?

Reply:
Yes, or we can make it `api` in core.